Reader.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+class Reader:
+
+    weathers = []
+    header_row = False
+
+    def read_file(self, name):
+        with open(name) as day:
+            if self.header_row:
+                next(day)
+            for record in day:
+                record = record.split(",")[0:10]
+                self.weathers.append(record)
+                self.header_row = True
+
+    def read_folder(self, path):
```

Comment:
Which folder is being read? Rename it to something descriptive.

Reply:
Are we just reading folder or file names in a folder? Both are two different things.